account_asset_management/security/ir.model.access.csv

```diff
@@ -1,15 +1,8 @@
 id,name,model_id:id,group_id:id,perm_read,perm_write,perm_create,perm_unlink
 access_account_asset_profile_invoice,account.asset.profile,model_account_asset_profile,account.group_account_invoice,1,0,0,0
-access_account_asset_profile_user,account.asset.profile,model_account_asset_profile,account.group_account_user,1,0,0,0
 access_account_asset_profile_manager,account.asset.profile,model_account_asset_profile,account.group_account_manager,1,1,1,1
 access_account_asset_invoice,account.asset,model_account_asset,account.group_account_invoice,1,1,1,1
```

Comment:
Should invoice users to have access to assets? I think it's better that only accountants have it.